tests/testUtils.py

```diff
@@ -20,11 +20,12 @@
 class Utils:
     Debug=False
     FNull = open(os.devnull, 'w')
-    EosClientPath="programs/eosioc/eosioc"
-    EosWalletPath="programs/eosio-walletd/eosio-walletd"
-    EosServerName="eosiod"
-    EosServerPath="programs/eosiod/%s" % (EosServerName)
+    EosClientPath="programs/eosc/eosc"
+    EosWalletPath="programs/eos-walletd/eos-walletd"
+    EosServerName="eosd"
+    EosServerPath="programs/eosd/%s" % (EosServerName)
     EosLauncherPath="programs/launcher/launcher"
```

Comment:
Looks like this diff went the wrong way. The removed lines are correct.

Reply:
This script is identical for master and noon branch. This section defines the master specific names. Method iAmNoon() sets the Noon specific names. I did this cause it was becoming a pain-point to sync the script between the two branches. Once noon becomes master we will get rid of this bridge code.